packages/components/src/LoginComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
```

Comment:
Is this component superseded by `dapp/src/components/Account/Auth/EmailAuth.tsx`? That one is mostly identical but with some additional bits, this one is only used in storybook.